packages/at_secondary_server/lib/src/server/at_secondary_config.dart

```diff
@@ -675,6 +688,19 @@ class AtSecondaryConfig {
     }
   }
 
+  static List<String> get protectedKeys {
+    try {
+      YamlList keys = getConfigFromYaml(['hive', 'protectedKeys']);
+      List<String> protectedKeys = [];
+      for (var key in keys) {
+        protectedKeys.add(key);
+      }
+      return protectedKeys.isEmpty ? _protectedKeys : protectedKeys;
```

Comment:
change this to
 protectedKeysFromConfig.isNotEmpty ? protectedKeysFromConfig : _defaultProtectedKeys;

Reply:
fixed as part of [567351b](https://github.com/atsign-foundation/at_server/pull/1471/commits/567351b6fe6c7c9af33b2644d579f4a616e8d71e)